packages/nomicfoundation-solidity-language-server/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: nomicfoundation-solidity-language-server
+description: Solidity language server by NomicFoundation
+homepage: https://github.com/NomicFoundation/hardhat-vscode
+licenses:
+  - MIT
+languages:
+  - Solidity
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/%40ignored/solidity-language-server@0.6.14
```

I might be missing something, but the `@ignored` scope doesn't seem to be an official one?

```suggestion
  id: pkg:npm/%40nomicfoundation/solidity-language-server@0.6.16
```

---

You are right, changed to oficial